ACE/ACEXML/apps/svcconf/ACEXML_XML_Svc_Conf_Parser.rc

```diff
@@ -0,0 +1,30 @@
+#include "..\..\..\ace\Version.h"
+
+1 VERSIONINFO
+ FILEVERSION ACE_MAJOR_VERSION,ACE_MINOR_VERSION,ACE_MICRO_VERSION,0
+ PRODUCTVERSION ACE_MAJOR_VERSION,ACE_MINOR_VERSION,ACE_MICRO_VERSION,0
+ FILEFLAGSMASK 0x3fL
+ FILEFLAGS 0x0L
+ FILEOS 0x4L
+ FILETYPE 0x1L
+ FILESUBTYPE 0x0L
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904B0"
+        BEGIN
+            VALUE "FileDescription", "ACE\0"
```

Comment:
Shouldn't be FileDescription and InternalName be different for each DLL?

Reply:
Updated. Thank you!